index.html

```diff
@@ -283,50 +283,69 @@
 
 ## People &amp; Data {#people-data}
 
-A <dfn>user</dfn> (also <dfn data-lt="individual">person</dfn> or <dfn>data subject</dfn>) is any natural
-person.
-
-We define <dfn data-lt="data">personal data</dfn> as any information relating to a [=person=] such
-that:
-
-* this [=person=] is identified, directly or indirectly, by reference to an
-  identifier such as a name, email address, an arbitrary identifier or
-  identification number, an online identifier such as an IP address or any
-  identifier attached to a device this [=person=] may be using, phone number,
-  location data, or factors specific to the physical, physiological, genetic,
-  mental, economic, cultural, or social identity of that [=person=], as well as
-  identifiers derived from such data, for instance through hashing; or
-* this [=person=] could reasonably be reidentified from a conjunction of this
-  data with other data; or
-* the data pertains to a group of people such that a [=person=] may find
-  themselves to be the subject of a treatment related to this group, even if the
-  entity carrying out the treatment has no way to identify that [=person=].
+A <dfn>user</dfn> (also <dfn data-lt="individual">person</dfn> or <dfn>data subject</dfn>)
+is any natural person.
+
+We define <dfn data-lt="data">personal data</dfn> as any information relating to a
+[=person=] such that:
+
+* The [=data=] references this [=person=] directly, for instance through their name,
+  email address, phone number, or other such traditional identifiers.
+* The [=data=] can identify this [=person=] through an arbitrary string or identification
+  number, an online identifier such as an IP address or any identifier attached to a
+  device which this [=person=] may be using.
+* The [=data=] is about a [=person=]'s location (or of a device that a [=person=] would use).
+* The [=data=] contains factors specific to the physical, physiological, genetic, mental,
+  economic, cultural, or social identity of a [=person=].
+* The [=data=] is comprised of identifiers derived from other types of data in this list,
+  for instance through hashing.
+* The [=person=] could reasonably be reidentified from a conjunction of this
+  [=data=] with other data.
+* The [=data=] relates to a group of people such that a [=person=] may be treated in a
```

Comment:
We're inconsistently precise here: we only include particular kinds of data about one person, but then we include any data at all about a group. Is there a definition of personal data that we could cite instead of inventing our own?

Reply:
You're right that there is an issue here, but I don't think it's that. What is listed here is anything that could count as identifying someone. Personal data should be that plus anything attached to identifying data. I'll make a fix.

There isn't a good unified definition of personal data that I know of. That's sort of the challenge (but also the usefulness) here: coming up with one that works!